calcom/cal.com
# MS Teams use `joinWebUrl` from response body

## What does this PR do?



This PR changes where we retrieve the MS Teams join URL from the response body. Before we were checking for the presence of `joinUrl`. Reading through forums and the docs sometimes `joinUrl` may not be present. In the docs for the `onlineMeeting` resource `joinUrl` is not defined but `joinWebUrl` is. When creating a Teams link both `joinUrl` and `joinWebUrl` are the same. 

https://learn.microsoft.com/en-us/graph/api/resources/onlineMeeting?view=graph-rest-1.0#properties

Fixes # (issue)



**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- Add MS Teams to your account
- Create a meeting
- The email & calendar event should contain a Teams URL

## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
